test/docker-compose.yml

```diff
@@ -415,15 +415,17 @@ services:
 
   zkevm-mock-l1-network:
     container_name: zkevm-mock-l1-network
-    image: hermeznetwork/geth-zkevm-contracts:v1.1.0-fork.4
+    image: hermeznetwork/geth-zkevm-contracts:v1.1.0-snapchain
```

Comment:
let's use https://hub.docker.com/r/snapchain/zkevm-contracts

what's your docker hub username? I can grant you push access:
![image](https://github.com/Snapchain/zkevm-node/assets/111917526/2d55893d-0e2c-45bb-a621-01ca270fc088)